mkfit-phase1-hltiter0.json

```diff
@@ -1,6 +1,6 @@
 {
- "m_iteration_index": 0,
- "m_track_algorithm": 4,
+ "m_iteration_index": 100,
+ "m_track_algorithm": 100,
```

Comment:
@mmasciov what's this change for?
(As far as I can tell it does not affect the current menu, is it purely technical to allow more iterations?)